core/src/scene/styleContext.cpp

```diff
@@ -487,6 +487,11 @@ void StyleContext::parseStyleResult(StyleParamKey _key, StyleParam::Value& _val)
                 _val = static_cast<uint32_t>(duk_get_uint(m_ctx, -1));
                 break;
             }
+            case StyleParamKey::angle: {
+                std::string value = doubleToString(static_cast<double>(duk_get_number(m_ctx, -1)));
+                _val = StyleParam::parseString(_key, value);
```

Comment:
Can't we just assign the double directly to the parameter value?

Reply:
Ohh yes, since `auto` return will be handled by `duk_is_string`.

Reply:
Updated!

Reply:
K actually fixed now!